BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsBaseWorker.java

```diff
@@ -14,16 +14,14 @@
  * This class is used internally by the SDK and should not be used directly.
  * It is not subject to semantic versioning and may change at any time.
  */
+// NEXT-MAJOR-VERSION: remove this class, it may no longer be needed
```

Comment:
Nitpick since I think we usually search for underscores
```suggestion
// NEXT_MAJOR_VERSION: remove this class, it may no longer be needed
```